### ISSUE TITLE
V3 deprovisions 

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#7363]
Per Maggie:

> I confirmed the following sites can be decommissioned:
- [x] mcallister.lab.uiowa.edu
- [x] prahlad.lab.uiowa.edu
- [x] fritzsch.lab.uiowa.edu (internal prod domain)
- [x] slusarski.lab.uiowa.edu (internal prod domain)

**Comment by e-marie-w** [comment]
I attempted to deprovision two of these today and could not, kept getting `[error] Unable to get current application with Drush.` message.

**Comment by joewhitsitt** [comment]
Is this with the `./vendor/bin/blt umd` command?

**Comment by e-marie-w** [comment]
Yes...I was wondering if the Drush 12 stuff might be causing this.

**Comment by joewhitsitt** [comment]
Just curious if you have done a `ddev composer install` since pulling main and the drush 12 stuff?

**Comment by e-marie-w** [comment]
Yes, I always try to after pulling main and looking back I can see I did.

**Comment by joewhitsitt** [comment]
I can confirm this is choking on the same kind of drush status task as https://github.com/uiowa/uiowa/issues/7354, but this time from https://github.com/uiowa/uiowa/blob/main/blt/src/Blt/Plugin/Commands/MultisiteCommands.php#L1083

**Comment by bspeare** [comment]
Is this blocked or do we need to create a new issue to investigate fixing this?

**Comment by joewhitsitt** [comment]
@bspeare we have the card https://github.com/orgs/uiowa/projects/20/views/1?filterQuery=drush&pane=issue&itemId=53434048 from planning yesterday. If you really wanted to push this forward, I think you could try downgrading similar to #7354 but I think there is an issue in returning the application name.

**Comment by joewhitsitt** [comment]
Based on https://github.com/uiowa/uiowa/pull/7382, I think this is unblocked
